src/DOMupdates.js

```diff
@@ -1,3 +1,114 @@
-{
-  
+import $ from 'jquery';
+
+let domUpdates = {
+  showError() {
+    $('#error-message').removeClass('hidden');
+  },
+  // --------------------------- Manager Functionality -----------------
+  loadManagerPage() {
+    $('#login-page').addClass('hidden').removeClass('visible');
```

Comment:
really like the double functionality of adding and removing a class simultaneously